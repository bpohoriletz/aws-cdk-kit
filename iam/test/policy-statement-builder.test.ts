import PolicyStatementBuilder from '../policy-statement-builder';

describe('new PolicyStatementBuilder()', () => {
  let instance: PolicyStatementBuilder;

  beforeEach(() => {
    instance = new PolicyStatementBuilder();
  });

  test('has defaults set', () => {
    expect(instance).toMatchSnapshot();
  });
});
