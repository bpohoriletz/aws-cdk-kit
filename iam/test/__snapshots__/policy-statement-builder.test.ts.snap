// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`new PolicyStatementBuilder() has defaults set 1`] = `
PolicyStatementBuilder {
  "policyStatementProps": PolicyStatementProduct {
    "actions": [],
    "resources": [],
  },
}
`;
